bitnami/grafana-operator/Chart.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: v2
+name: grafana-operator
+description: Kubernetes Operator based on the Operator SDK for creating and managing Grafana instances and dashboards.
+version: 1.0.2
+appVersion: 3.6.0
+
+keywords:
+  - grafana
+  - operator
+  - monitoring
+home: https://github.com/integr8ly/grafana-operator
+icon: https://bitnami.com/assets/stacks/grafana/img/grafana-stack-220x234.png
+sources:
+  - https://github.com/integr8ly/grafana-operator
+  - https://github.com/bitnami/bitnami-docker-grafana-operator
+
+maintainers:
+  - name: cellebyte
+    url: cellebyte@gmail.com
+  - name: Bitnami
+    email: containers@bitnami.com
+
+annotations:
+  category: Analytics
+
+dependencies:
+  - name: common
+    version: 0.x.x
```

There is a new major version for the common subchart
```suggestion
    version: 1.x.x
```